gems/sorbet-runtime/lib/types/props/serializable.rb

```diff
@@ -261,11 +261,16 @@ def add_prop_definition(prop, rules)
   end
 
   def message_with_generated_source_context(error, generated_method, generate_source_method)
-    generated_method = generated_method.to_s
-    line_loc = error.backtrace_locations.find {|l| l.base_label == generated_method}
-    return unless line_loc
+    line_label = error.backtrace.find {|l| l.end_with?("in `#{generated_method}'")}
```

Comment:
If you are open to it, I'd like to suggest to first try to use `backtrace_locations` if it's present, and then only fallback.

This way you get the speed benefit in the vast majority of cases, and only degrade to string parsing in a minority of cases.

Also this `l.end_with?("in `#{generated_method}'")` won't match on Ruby 3.4 anyway, so I'll open another PR.